addons/stock_landed_costs/res_config_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
NotImportant: Rename the file to purchase_config_settings_views.xml .

By the way, use <odoo> instead of <openerp><data> .